core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/api/component/IconIdProvider.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2011-2018 B2i Healthcare Pte Ltd, http://b2i.sg
```

Comment:
New files should indicate only the current year in their license header. Update it, please!

Reply:
updated.